src/main/java/core/mvc/tobe/support/RequestBodyArgumentResolver.java

```diff
@@ -0,0 +1,39 @@
+package core.mvc.tobe.support;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import core.annotation.web.RequestBody;
+import core.mvc.ObjectMapperException;
+import core.mvc.tobe.MethodParameter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.stream.Collectors;
+
+public class RequestBodyArgumentResolver extends AbstractAnnotationArgumentResolver {
+    private static final Logger logger = LoggerFactory.getLogger(RequestBodyArgumentResolver.class);
+
+    private static final ObjectMapper objectMapper = new ObjectMapper();
+    private static final String NEWLINE = "\n";
+
+    @Override
+    public boolean supports(MethodParameter methodParameter) {
+        return supportAnnotation(methodParameter, RequestBody.class);
+    }
+
+    @Override
+    public Object resolveArgument(MethodParameter methodParameter, HttpServletRequest request, HttpServletResponse response) {
+        try {
+            String body = request.getReader()
+                    .lines()
+                    .collect(Collectors.joining(NEWLINE));
+
+            return objectMapper.readValue(body, methodParameter.getType());
```

Comment:
`Reader` 를 직접 넘겨줘도 될 것 같아요 :)
```suggestion
            return objectMapper.readValue(request.getReader(), methodParameter.getType());
```

Reply:
감사합니다 :)